packages/mds-jurisdiction-service/ormconfig.ts

```diff
@@ -17,4 +17,5 @@
 import { JurisdictionRepository } from './server/repository'
 
 // Make connection options available to TypeORM CLI
-module.exports = JurisdictionRepository.config
+const { ormconfig } = JurisdictionRepository
```

Comment:
I think you can do something like `import { JurisdictionRepository: { ormconfig } } from './server/repository'`

Reply:
JK, you can't do nested destructuring for imports... 👎